http/vibe/http/client.d

```diff
@@ -173,6 +134,38 @@ private bool isTlsNeed(in URL url, in HTTPClientSettings settings)
 	return use_tls;
 }
 
+//TODO: remove @trusted
+private void httpRequesterDg(scope HTTPClientRequest req, in URL url, in HTTPClientSettings settings, scope void delegate(scope HTTPClientRequest req) requester) @trusted
```

Comment:
If the `delegate` aren't `@safe`, this can't be marked `@trusted`, ever

Reply:
This is already tagged as "Outdated".

I used @trusted temporary to not to make a mistake by making a big change right away

Reply:
In that case you could add `@system` to `httpRequesterDg` instead. `@trusted` is bad, because it communicates that the contained code has been manually verified to be safe, which is of course not the case with a user supplied callback.

Of course the next step at some point will be to templatize the callback parameter and let the compiler infer the safety of the function (which currently requires putting it in a different scope or replacing the `@safe:` at the beginning of the file).